www/runtest.php

```diff
@@ -1311,6 +1313,17 @@ function ValidateParameters(&$test, $locations, &$error, $destination_url = null
 
             if( !$test['aftEarlyCutoff'] && $settings['aftEarlyCutoff'] )
                 $test['aftEarlyCutoff'] = $settings['aftEarlyCutoff'];
+
+            if ( $test['lighthouse'] ) {
+              $lh_error = '';
+              if ( strpos($test['browser'], 'Chrome') === FALSE )

```

Comment:
Didn't see any setting that reveals if the agent is node or not, anything I missed?